modules/controller_utils/__init__.py

```diff
@@ -11,9 +11,6 @@
 # Root directory of the specification of the Arena (and match)
 ARENA_ROOT = Path(os.environ.get('ARENA_ROOT', REPO_ROOT.parent))
```

Comment:
While this wasn't added here we are now recommending its use, setting ARENA_ROOT as an environment variable only works when setting an absolute path. If a relative path is used every file takes that path relative to itself

Reply:
This change probably should be its own PR

Reply:
See #266

Reply:
Ah, that's a shame. Ultimately absolute paths are what is needed here, so I'm not inclined to change this. We could add a warning if we detect the user has passed a relative path though.